### ISSUE TITLE
Doesn't work with keyboard navigation [accessibility,a11y]

### ISSUE THREAD

**Issue by missmatsuko** [benitolopez/hotel-datepicker#55]
Part of the problem is that a lot of the event triggers are things other than buttons.

Expected:
- Should be able to open the datepicker using only the keyboard (maybe when the text input is focused?)
- Should be able to change the date using only the keyboard. This can maybe be fixed by adding buttons in each date cell and replacing the month arrow spans with buttons, and moving the event handlers to those button elements.

**Comment by benitolopez** [comment]
Sorry for the delay but I'm very busy. I will try to add some keyboard triggers. I don't think that we need to involve the month arrows. We can just use the days to navigate a day forward/backward (left/right arrow keys) and a week forward/backward with the up/down keys. Of course, the input should be used to open the calendar.

**Comment by benitolopez** [comment]
After more than four years (sorry), the version 4.5.0 introduces accessibility improvements and keyboard navigation.
